pkg/namespaces/namespaces.go

```diff
@@ -337,7 +337,7 @@ type NetworkMode string
 
 // IsNone indicates whether container isn't using a network stack.
 func (n NetworkMode) IsNone() bool {
-	return n == noneType
+	return n == noneType || n == ""
```

Comment:
Is this accurate? "" could also be the default (bridge), couldn't it?

Reply:
The thing is "none" is never set via the `With...` functions so all we have is an empty string for the NetMode.
Bridge and slirp4netns are set via `WithNetNS` which should set the mode correctly. There is a lot of code using `NetMode.IsBridge()` which only matches `bridge` so I think this is correct.

Reply:
Alright. We really ought to make this more reliable in 4.0, having some cases not assign this is iffy.

Reply:
OK, this might be a bigger problem than I thought. If a container is joined to a pod or another container it also has an empty NetworkMode.